assets/component-volume-pricing.css

```diff
@@ -0,0 +1,28 @@
+volume-pricing {
+  display: block;
+  margin-top: 2rem;
+}
+
+volume-pricing li:nth-child(even) {
+  background: rgba(var(--color-foreground), 0.03);
```

Comment:
> We do use .03 in a couple places, but .04 definitely seems more common in the codebase. Just making sure we really want/need that specific value.

@sofiamatulis @kmeleta I'm totally okay with changing this to `.04` if that's more consistent and common with current Dawn conventions + the current codebase.